scripts/translate_transcript.py

```diff
@@ -2,65 +2,78 @@
 import argparse
 import asyncio
 import json
+from pathlib import Path
 from unittest.mock import AsyncMock
 
 from mentat.code_file_manager import CODE_FILE_MANAGER, CodeFileManager
 from mentat.git_handler import GIT_ROOT
 from mentat.parsers.block_parser import BlockParser
+from mentat.parsers.git_parser import GitParser
 from mentat.parsers.parser import Parser
 from mentat.parsers.replacement_parser import ReplacementParser
 from mentat.parsers.split_diff_parser import SplitDiffParser
 from mentat.parsers.unified_diff_parser import UnifiedDiffParser
 from mentat.session_stream import SESSION_STREAM
 from mentat.utils import convert_string_to_asyncgen
 
-CODE_FILE_MANAGER.set(CodeFileManager())
-SESSION_STREAM.set(AsyncMock())
-
-
 parser_map: dict[str, Parser] = {
     "block": BlockParser(),
     "replacement": ReplacementParser(),
     "split-diff": SplitDiffParser(),
     "unified-diff": UnifiedDiffParser(),
+    "git": GitParser(),
```

Comment:
I don't really think it should be added to this map; it isn't a parser, we don't have a prompt for it, and the user can't select it

Reply:
Oh whoops, somehow I completely missed what file this was in! Ignore this haha